packages/share/lib/share.dart

```diff
@@ -14,9 +15,24 @@ const MethodChannel _kChannel = const MethodChannel('plugins.flutter.io/share');
 /// It uses the ACTION_SEND Intent on Android and UIActivityViewController
 /// on iOS.
 ///
+/// The optional `sharePositionOrigin` parameter can be used to specify a global
+/// origin rect for the share sheet to popover from on iPads. It has no effect
+/// on Android or on phones.
```

Comment:
"on Android or on phones" sounds as if Android devices are not real phones. Also: iPod touch is neither an iPad nor a phone.

Reply:
Reworded